### DISCUSSION TITLE
can I use GitHub Copilot Labs in "Visual Studio",not in "Visual Studio Code"

### DISCUSSION THREAD

**Discussion by Sybiil** [community/community#46693] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I find  GitHub Copilot Labs extension only in Visual Studio Code,and GitHub Copilot  extension in Visual Studio.
Is there any way to use GitHub Copilot Labs in Visual Studio?

**Comment by jge162** [comment, marked as answer]
Hello @Sybiil currently GitHub Copilot Labs is only available as a Visual Studio Code extension, and there is no support for Visual Studio that I can see. Thanks for the post.
